packages/babel-standalone/src/overrideModuleResolution.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * babel-loader causes problems as it's not part of the monorepo. It pulls in
+ * an older version of babel-core (the version referenced by the root
+ * package.json), rather than the version of babel-core that's in the repo. The
+ * only way to solve this without moving babel-loader into the monorepo is to
+ * override Node's module resolution algorithm to specify a custom resolver for
+ * babel-core to *force* it to use our version.
+ *
+ * Here be dragons.
```

Comment:
I'm always worried that inserting Emoji into code is going to break something somewhere 😛